config/storybook/styles/_storybook-global.scss

```diff
@@ -65,3 +65,7 @@ h6 {
     transform: translate3d(0, 64px, 0);
   }
 }
+
+.cc-accordion__button {
```

Comment:
without this this button in storybook has Arial font. @creido-welly, is it tokay for this to be here, or should I move it to `_accordion.scss`

Reply:
There's obviously a global style applied to the site which isn't present in SB. Try adding it to the accordion itself and see if that also fixes it in storybook.

Reply:
I added `font-family: inherit` to the accordion-button - thanks to this the correct font is used in storybook (Helvetica Neue) and for Windows users (Inter).